dashboards-observability/public/components/explorer/home_table/history_table.tsx

```diff
@@ -40,44 +40,100 @@ export function Table(options: TableData) {
   };
 
   const columns = [
+    {
+      field: 'type',
+      name: '',
+      width: '3%',
```

Comment:
does a percentage width work well for an icon? could you try zoom in and zoom out the page to see if it still looks good

Reply:
the width is for the distance between two columns. I tried zoom in and out, still looks good. I have attached an image for reference
<img width="1792" alt="new view" src="https://user-images.githubusercontent.com/18598378/140162738-62b3b029-af5f-4e76-920f-dc8b3a66de9b.png">
.